content/reference/agent-configuration.md

```diff
@@ -22,6 +22,101 @@ As the communication bridge between the Rhize Data Hub and your plant, the agent
+| `password`         | The password credentials to authenticate with the OIDC server. |
+
+## OpenTelemetry
+
+| Attribute | Description |
+|---------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `serverUrl`         | The URL of the OpenTelemetry server.  <br />(Default: `localhost:4317`) |   
+
+## OPCUA 
+
+| Attribute | Description |
+|---------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `DiscoveryUrl`         | The URL to locate and connect to OPC UA servers on a network. <br />(Default: `opc.tcp://localhost:4840`) |
+| `Endpoint`         | The URL of the OPC UA service server.  <br />(Default: `opc.tcp://localhost:4840`) |
+| `Username`         | The username credentials to authenticate with the OP CUA server. |
+| `Password`         | The password credentials to authenticate with the. |
+| `Mode`         | The operational mode of the OPC UA server/client.  <br />(Default: `None`) |
+| `Policy`         | The security measures for OPC UA server communication.  <br />(Default: `None`) |
+| `Auth`         | The authentication mechanisms and user access control.  <br />(Default: `Anonymous`) |
+| `AppUri`         | The application's unique URI within the OPC UA system.  <br />(Default: `opc.tcp://localhost:4840`) |
+
+## BUFFERS
+
+| Attribute | Description |
+|---------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `ProtocolQueueType`         | The type of queue used for buffering communication protocol data.  <br />(Default: `0`) |   
+
+## HEALTH
+
+| Attribute | Description |
+|---------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `PollInterval`         | The frequency of scans for component status and health.  <br />(Default: `1000`) |   
+| `SubscriptionTimeout`         | The maximum duration for waiting to receive updates from subscribed data sources.  <br />(Default: `60000`) |   
+| `SubscriptionMaxCount`         | The maximum number of concurrent subscriptions for monitoring.  <br />(Default: `5`) |   
+
+## MQTT
+
+| Attribute | Description |
+|---------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `Version`         | The version of MQTT used: `5.0` or `3.1.1`. <br />(Default: `3.1.1`) |   
```

Comment:
@Jarrah-libremfg Does the agent accept 5.0 yet?

Reply:
From what we're aware of, there is a MqttAdapterV5.go file implementation in progress, and the code reads both 3.1.1 and 5.0 versions, but the 5 case just raises an unsupported message currently.